### ISSUE TITLE
LARGE tooltip

### ISSUE THREAD

**Issue by Seeker9889** [desktop/desktop#2976]
### Description
Tooltip is huge. May be intentional, but it's not what I expected it to be.

### Version
**GitHub Desktop version:** 1.0.3

**OS version:** Microsoft Windows [Version 10.0.15063]

### Steps to Reproduce

1. GitHub Desktop is open IN AN EXTENDED SCREEN from my laptop (screen is larger than my desktop screen)
2. GitHub Desktop window is entirely on 2nd screen
3. Hover over Current Branch button

**Expected behavior:** A nice, normal tooltip appears on mouse hover over of Current Branch button

**Actual behavior:** Jabba the Tooltip appears on mouse hover over of Current Branch button

**Reproduces how often:** 100%

#### Additional Information

Hopefully it goes without saying: this is not a problem at all. I'm reporting because it seems like unexpected behavior

![bigtooltip](https://user-images.githubusercontent.com/28899942/31284203-82e7a912-aa7d-11e7-947a-558eb7e5e2ce.png)

**Comment by Seeker9889** [comment]
Laptop screen resolution: 3840 x 2160
2nd screen resolution: 2560 x 1440

**Comment by niik** [comment]
hey @Seeker9889. Jabba The ToolTip is not expected behavior as far as @github/design is concerned. That said I don't think we have a lot of influence over this. Is it possible that you switch between different DPI scaling modes on these two screens? That setup looks very similar to what I've got with my 4k screen on my laptop running at 200% DPI and my big external monitor running at 100%.

I'm asking because sometimes when running under such a setup Windows will get confused and render native components (such as the tooltip) in the DPI that was in use when the current logon session was established. So if you sign in on you laptop before attaching your external monitor some native components will be stuck at 200%.

**Comment by Seeker9889** [comment]
Yes, the scaling is different at about that level (i.e. 200-ish% on the laptop and 100% on the screen). I messed with the scaling to check, and the tooltip became a more relatively-normal size. So, it looks like the huge tooltip is because of the scaling. At least I know; thanks!

**Comment by tidy-dev** [comment]
Closed as we use [custom tooltips ](https://github.com/desktop/desktop/pull/13116) now and this seems to be a user screen configuration issue anyways/out of scope for Desktop.
